app/models/user.rb

```diff
@@ -387,6 +388,10 @@ def country
     Country.find_by_iso2(country_iso2)
   end
 
+  def newcomer?
+    person.nil? || wca_id.start_with?(Time.current.year.to_s)
+  end
```

Comment:
This is a little bit ambiguous definition of the term "Newcomer".
In the intuitive sense, a newcomer only denotes a person who has never competed before. The second part after the `||` is specific to WCA Newcomer Month. Can we rename the method or maybe split it in two?

Reply:
Rather than a comment, you should rename this method. If the name is `newcomer?` then there is **definitely** a strong intuitional bias towards "has never competed before".

The first thing that comes to mind is `newcomer_month_eligible?`